src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -137,28 +133,242 @@ public static String getProgressBar(Track track) {
                 " " + GeneralUtils.percentageFormat.format(percentage) + "%";
     }
 
-    private static char getPrefix(Guild guild) {
-        return guild == null ? Constants.COMMAND_CHAR : FlareBotManager.instance().getGuild(guild.getId()).getPrefix();
+    /**
+     * Gets a String stacktrace from a {@link Throwable}
```

Comment:
We really should limit this also to get the actual required info... eh whatever. Anyway, state what this actually returns and why we do this.